src/commands/contents.js

```diff
@@ -22,15 +21,18 @@ exports.builder = yargs => {
 			type: 'string',
 			describe:
 				'GitHub personal access token. Generate one from https://github.com/settings/tokens'
+		})
+		.option('limit', {
+			required: false,
+			type: 'number',
+			describe: 'limit the number of results to return to this number'
```

Comment:
It's not really about the results. It's more about the input repositories

```suggestion
			describe: 'limit the number of repositories to search for'
```

?